sympy/simplify/radsimp.py

```diff
@@ -989,7 +995,7 @@ def handle(expr):
 
 def rad_rationalize(num, den):
     """
-    Rationalize num/den by removing square roots in the denominator;
+    Rationalize ``num``/``den`` by removing square roots in the denominator;
```

Comment:
This is already the pythonic expression, so it should better be
```
``num/den``
```

Reply:
Changed.